### ISSUE TITLE
Metadata under original_image

### ISSUE THREAD

**Issue by jameelmoses** [joppuyo/acf-image-aspect-ratio-crop#31]
I know we have access to the original metadata under the `original_image`, but it would be nice if alt text, title and captions specifically could be copied over to the new image. Thoughts?

**Comment by joppuyo** [comment]
Yeah, I think this can be added as an option. Users just need to be aware that copying is a one-time thing that occurs at cropping time and all crops have independent metadata.

I'll look into adding this in a future version of the plugin.

**Comment by benvoynick** [comment]
I'd like to request that, if added, this be a setting that can be turned off and is off on existing installs by default. We're using logic to fall back to the original image for certain metadata if it's empty on the crop, and this functionality would get in the way of allowing updates on the original to affect existing crops.

**Comment by jameelmoses** [comment]
The more I've thought about it, and as @benvoynick pointed out, this could actually cause issues for the sake of making templating slightly cleaner. Probably best to leave as is because you always have `original_image` to access.

**Comment by mbateam** [comment]
How close are you to the upgrade being ready for beta testing?  I am debating whether I should hire a developer to create a separate plugin. However, I would rather contribute to an already successful product if you are open to doing a little custom work for payment of course.

**Comment by joppuyo** [comment]
I'm reopening this since I'm committed to shipping this feature in version 7.0 of the plugin. This will be of course an optional feature.

@mbateam I'm planning to ship the new major version of the plugin during this fall. I know progress has been a bit slow but the new version is almost finished, just some features need to be completed and I need to do some more testing.

Since I work on this plugin in my free time I'm a bit hesitant to make any kind of financial arrangement that involves a deadline. But if you want to support the plugin I have a [GitHub Sponsors account](https://github.com/sponsors/joppuyo) and I appreciate any contributions!

If this feature is something you need right now it might be best to hire a developer to implement this. Nevertheless, I plan to implement this feature in the near term but as I said, I can't give any definite deadline unfortunately.

**Comment by mbateam** [comment]
The only feature I want is an on-page cropping feature edit instead of a popup mode because it is easier for mobile devices. I assume a third party developer can do this.

**Comment by joppuyo** [comment]
@mbateam I think ticket #114 is the right place to discuss this feature instead of this one.

Anyway, the core idea of this plugin is to extend the basic ACF image field so that it enforces a certain aspect ratio.

I'm not planning on adding any kind of features beyond this, like integration with any kind of front-end page builder or anything like that, apart from ACF forms which use the same kind of API as ACF fields in the admin.

So, if you have very specific requirements it's best to hire a developer to create a plugin that fits your particular use case.
